conf/application.conf

```diff
@@ -87,19 +87,6 @@ play.i18n.langs = ["en"]
 # !!!WARNING!!! DO NOT CHANGE THIS ROUTER
 play.http.router = prod.Routes
 
-# Logger
-# ~~~~~
-# You can also configure logback (http://logback.qos.ch/), by providing a logger.xml file in the conf directory .
```

Comment:
why is all this gone?

Reply:
The compiler was giving warnings saying that any logging config in the .conf will be ignored and we should use a logback file, which we already are.